### ISSUE TITLE
Create Xpress techniques and assign instruments and users on dev/prod

### ISSUE THREAD

**Issue by simonfernandes** [UserOfficeProject/user-office-project-issue-tracker#1194]
We need to create techniques and assign instruments and users to them ready for the Xpress release. Having them in place will also enable better testing.

**Comment by simonfernandes** [comment]
Hi @Cat-Lucifer. This would be really useful to set up soon, though I don't think as developers we know which techniques should be created and which instruments they'll have. Is this something you'd either have time to create or provide the data for?

**Comment by Cat-Lucifer** [comment]
@simonfernandes  Yep, I can do this.  I will need to double check with GLs that we want to duplicate those already in Direct, but that shouldn't hold this up.  When creating, are there any rules I need to abide by for the technique short code?

![image](https://github.com/user-attachments/assets/61734af0-b17d-46bc-b98e-e5a410a260fa)

**Comment by simonfernandes** [comment]
@Cat-Lucifer Thanks very much. 

What did you mean by duplicate those already in Direct? If you meant instruments then we can re-use the existing ones and simply assign them to techniques. 

For the technique short code it should be fine to make it the same as the name, to copy what we've done with instruments.

We will also need to create an ALF instrument as previously mentioned.

**Comment by Cat-Lucifer** [comment]
@simonfernandes By duplicate from Direct, I meant that I'm assuming we are going to use the same techniques that are already listed in the Direct template (Q key `scientific_technique`).  There are no instruments associated with these techniques, and that's where I will need to work with the GLs.

**Comment by simonfernandes** [comment]
Thanks for creating these on prod @Cat-Lucifer, and I understand they're still subject to change over the coming weeks. I hope you don't mind but I added the [four other instruments](https://github.com/UserOfficeProject/issue-tracker/issues/87#issuecomment-2417341926) and have assigned them to the techniques mentioned. I assigned you as an instrument contact as I wasn't sure who this should be, and they don't have scientists assigned to them, but neither matters for Xpress purposes.

**Comment by simonfernandes** [comment]
I've also noticed when testing the legacy import that `HRPD` isn't attached to a technique. Is because it's not accepting proposals for the next few years? We could still attach it to a technique (to enable the legacy import), but omit it from the list of instruments attached to the call.

**Comment by Cat-Lucifer** [comment]
@simonfernandes

Yes sorry, I completely forgot about those four other instruments - thanks for adding them in.  I think it's fine that I'm listed as contact.  We can change later if need be.

Yes, HRPD is not being offered right now (and won't be for a few years), but I have now added it to the relevant technique.

For Xpress, won't I be adding techniques to the call?  Or, will I be adding in instruments as normal and they will pull across the technique associated?

**Comment by simonfernandes** [comment]
@Cat-Lucifer Thanks for that. For Xpress, we'll just be attaching instruments to the call, and from that it will decide which techniques to offer. It should allow us to keep unavailable instruments attached to techniques and thus not break viewing permissions on older proposals.

**Comment by simonfernandes** [comment]
Hi @Cat-Lucifer, there's no rush but please could you let us know when you have spoken with group leaders and finalised techniques and instruments? I couldn't find where you mentioned the timeframe but I think you said those comms would run into December?

@mehta-pooja123

**Comment by bashanlam** [comment]
Sprint note (11/12)
@Cat-Lucifer we need this before 10-Feb-25.

**Comment by mehta-pooja123** [comment]
@Cat-Lucifer Another bump for this information

**Comment by Cat-Lucifer** [comment]
@mehta-pooja123 @bashanlam Discussed with science group leaders today.  They will have a deadline of Thursday 23 January to confirm the instruments already assigned and provide a staffing list, and Xpress questionnaire updates.

**Comment by mutambaraf** [comment]
Sprint 04/02/2025 : We need data for dev.

**Comment by Cat-Lucifer** [comment]
Techniques, instruments and associated instrument scientists all finalised in prod.
